src/main/java/by/teachmeskills/springboot/dao/orm/books/BooksOrmDAO.java

```diff
@@ -0,0 +1,26 @@
+package by.teachmeskills.springboot.dao.orm.books;
+
+import by.teachmeskills.springboot.dao.books.BooksDAO;
+import by.teachmeskills.springboot.dao.orm.OrmAbstractDAO;
+import by.teachmeskills.springboot.entity.book.BookEntity;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public class BooksOrmDAO extends OrmAbstractDAO<BookEntity> implements BooksDAO {
+    @Override
+    protected Class<BookEntity> getEntityClass() {
+        return BookEntity.class;
+    }
+
+    @Override
+    public Optional<BookEntity> getByTitle(String title) {
+        return null;
```

Comment:
If smth is not implemented - don't return null, throw _NotImplementedException_ instead